src/plugins/analysis/cve_lookup/code/cve_lookup.py

```diff
@@ -142,11 +144,11 @@ def generate_search_terms(product_name: str) -> list[str]:
 
 def find_matching_cpe_product(cpe_matches: list[Product], requested_version: str) -> Product:
```

Comment:
Am I missing something or don't we convert the `requested_version`, too? If not, shouldn't we do so in order to establish a somewhat consistent comparability between requested version and the `next_closest_version`?

Reply:
I replaced `parse_version` with the new function here now, too